lib/active_model_serializers/model.rb

```diff
@@ -30,6 +30,8 @@ def updated_at
     end
 
     def read_attribute_for_serialization(key)
+      return send(key) if respond_to?(key)
```

Comment:
FWIW you *can* fix the current class with something like [this](https://github.com/richmolj/active_model_serializers/blob/45043f29ba743ae9390aefc8965edebc45e08960/lib/active_model_serializers/model.rb#L13-L17) and [this](https://github.com/richmolj/active_model_serializers/blob/45043f29ba743ae9390aefc8965edebc45e08960/lib/active_model_serializers/model_mixin.rb#L17). That got the tests passing, including the scenario above, though it's just POC code.